web/server/vue-cli/src/components/Report/ReportFilter/Filters/DetectionStatusFilter.vue

```diff
@@ -14,6 +14,36 @@
     </template>
 
     <template v-slot:append-toolbar-title>
+      <tooltip-help-icon>
+        Filter reports by the <b>latest</b> detection status.<br><br>
```

Comment:
Shouldn't `<strong>` be used in the newest HTML standards to convey semantics?

Also, `<br />`.